commit 3d7ef13942b847452c5bfdc70afb30022ecc8d65 (refs/changes/76/4176/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-20T10:35:51-07:00 (1 year, 4 months ago)
    
    TRITON-475 move core images from sdc-minimal-multiarch + pkgs to triton-origin for (much) smaller size (manta-reshard)

diff --git a/targets.json.in b/targets.json.in
index ef451e5..7021e7f 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1429,11 +1429,9 @@ cat <<EOF
     "image_name": "manta-reshard",
     "image_description": "Manta Resharding System",
     "image_version": "1.0.0",
-    "// image_uuid": "sdc-minimal-multiarch-lts@15.4.1",
-    "image_uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-    "pkgsrc": [
-      "coreutils-8.23nb2"
-    ],
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
+    "pkgsrc": [],
     "repos": [
         {"url": "git@github.com:joyent/manta-reshard.git"}
     ],
